oauth2_provider_jwt/views.py

```diff
@@ -19,6 +19,7 @@ def _get_access_token_jwt(self, request, expires_in):
         if payload_enricher:
             fn = import_string(payload_enricher)
             extra_data = fn(request)
+            extra_data['username'] = request.POST.get('username')
```

Comment:
Every request won't have a username. For example on `refresh_token` actions. It should not be inside this `if payload_enricher` and something like:

```
if request.POST.get('username'):
    extra_data['username'] = request.POST.get('username')
```